<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfinityEdge</class>
 <widget class="QMainWindow" name="InfinityEdge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infinity Edge</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cmdGenerate">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdRandomize">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>420</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Randomize</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidRand">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>350</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>355</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Randomization</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWidth">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>390</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>393</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>392</y>
      <width>10</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHeight">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>390</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidEmpty">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>320</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>325</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>295</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Corners</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidCorners">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>263</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crosses</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidCross">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>233</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ends</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidEnd">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>203</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangles</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidTriangles">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>541</x>
      <y>174</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lines</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidLine">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRand">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>146</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Direct Randomization</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
